test/typecheck/main.go

```diff
@@ -41,12 +41,15 @@ import (
 )
 
 var (
-	verbose   = flag.Bool("verbose", false, "print more information")
-	cross     = flag.Bool("cross", true, "build for all platforms")
-	platforms = flag.String("platform", "", "comma-separated list of platforms to typecheck")
-	timings   = flag.Bool("time", false, "output times taken for each phase")
-	defuses   = flag.Bool("defuse", false, "output defs/uses")
-	serial    = flag.Bool("serial", false, "don't type check platforms in parallel")
+	verbose    = flag.Bool("verbose", false, "print more information")
+	cross      = flag.Bool("cross", true, "build for all platforms")
+	platforms  = flag.String("platform", "", "comma-separated list of platforms to typecheck")
+	timings    = flag.Bool("time", false, "output times taken for each phase")
+	defuses    = flag.Bool("defuse", false, "output defs/uses")
+	serial     = flag.Bool("serial", false, "don't type check platforms in parallel")
+	skipTest   = flag.Bool("skip-test", false, "don't type check test code")
+	tags       = flag.String("tags", "", "comma-separated list of additional build tags to apply")
+	ignoreDirs = flag.String("ignore-dirs", "", "comma-separated list of directories to ignore")
```

Comment:
clarify things specified here are in addition to standard ignored dirs

Reply:
done